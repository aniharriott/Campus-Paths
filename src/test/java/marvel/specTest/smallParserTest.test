# Test parser on smallTest.tsv

LoadGraph small smallTest.tsv

ListChildren small a