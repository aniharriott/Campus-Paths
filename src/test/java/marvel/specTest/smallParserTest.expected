# Test parser on smallTest.tsv

loaded graph small

the children of a in small are: b(one)